File: check_columns.py
from import_bank_xml_data import get_db_connections

conn1, conn2 = get_db_connections()
cur = conn2.cursor()
cur.execute("SELECT column_name FROM information_schema.columns WHERE table_name = 'bog_gel_raw_893486000' ORDER BY ordinal_position")
cols = cur.fetchall()
print("Columns in bog_gel_raw_893486000:")
for c in cols:
    print(f"  {c[0]}")
conn2.close()
conn1.close()

File: check_columns.js
const { PrismaClient } = require('@prisma/client');
const prisma = new PrismaClient();

async function checkColumns() {
  try {
    const result = await prisma.$queryRaw`
      SELECT column_name, data_type 
      FROM information_schema.columns 
      WHERE table_name = 'consolidated_bank_accounts' 
      AND column_name IN ('exchange_rate', 'correction_date')
      ORDER BY column_name
    `;
    
    console.log('Columns in consolidated_bank_accounts:');
    console.log(result);
  } catch (error) {
    console.error('Error:', error.message);
  } finally {
    await prisma.$disconnect();
  }
}

checkColumns();
